tests/integration/models/device.spec.coffee

```diff
@@ -412,6 +412,24 @@ describe 'Device Model', ->
 				promise = resin.models.device.isOnline(999999)
 				m.chai.expect(promise).to.be.rejectedWith('Device not found: 999999')
 
+		describe.only 'resin.models.device.isActive()', ->
```

Comment:
Don't commit `.only`! This disables the whole of the rest of the SDK test suite.

It's easy to miss ofc. Good opportunity for a shameless plug: https://github.com/pimterry/git-confirm (with `git config --add hooks.confirm.match ".only"`)